src/css/map-view.css

```diff
@@ -1323,7 +1323,7 @@ other class: .ui-slider-range */
     border: 1px solid var(--portal-col-bkg-active);
     border-radius: 4px;
     box-sizing: border-box;
-    color: var(--map-col-text);
+    color: var(--map-col-text-body);
```

Comment:
--map-col-text-body is only defined when a theme is used, while some portals/main repos don't use themes (you can test on ADC or DRP). do we need a fallback color for those use cases?

same question for other color variables.